content/yaml-quick-start/relocating from AppCenter to Codemagic.md

```diff
@@ -0,0 +1,718 @@
+{{</markdown>}}
+{{< /tab>}}
+
+{{< tab header="Webhooks with Gitlab" >}}
+{{<markdown>}}
+Navigate to Settings > Webhooks, paste the payload URL and check the following boxes in the Trigger section: Push events, Tag push events, Merge request events. Also, be sure to enable SSL verification.
+{{</markdown>}}
+{{< /tab>}}
+
+{{< tab header="Webhooks with Bitbucket" >}}
+{{<markdown>}}
+Open your application repository, go to Settings > Webhooks (in Workflow section) > Add webhook, then enter an arbitrary title for the webhook and paste the payload URL in the URL field. For Triggers, select Choose from a full list of triggers and select the following events: Push in the Repository section and Created, Updated, Merged in the Pull Request section.
+{{</markdown>}}
+{{< /tab>}}
+
+{{< tab header="Webhooks with AWS CodeCommit" >}}
+{{<markdown>}}
+To start using webhooks with AWS CodeCommit, it is first necessary to create a subscription with the AWS Simple Notification Service.
+
+### Configure Subscription
+
+1. Open up AWS Simple Notification Service in the AWS Console.
+2. Navigate to Topics > Create topic.
+3. Set the type to Standard, give the topic a name and click on Create topic.
+4. Navigate to Subscriptions > Create subscription.
+5. Select the previously configured topic, set the protocol to HTTPS, and set the Codemagic payload URL as the endpoint.
+6. Confirm that Enable raw message delivery is unticked.
+7. Proceed by clicking Create subscription.
+8. In the Codemagic UI, navigate to your application and select the Webhooks tab.
+9. Under Recent deliveries, choose the most recent webhook, and copy the subscription link under the Results tab to your browser.
+
+### Configure Webhook Events
+
+Open your application repository and navigate to Notify > Create notification rule and enter a name for your Notification rule.
+
+Under Events that trigger notifications, select the Source updated and Created events in the Pull request section and the Created and Updated events in the Branches and tags section.
+
+Set the target type to SNS topic, select a configured target and click on Submit.
+
+If, after triggering a build, the SNS Notification target status shows as Unreachable, navigate to the topic settings and modify the access policy to match the following structure:
+
+{{< highlight json "style=paraiso-dark">}}
+{
+  "Version": "2008-10-17",
+  "Statement": [
+    {
+      "Sid": "CodeNotification_publish",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "codestar-notifications.amazonaws.com"
+      },
+      "Action": "SNS:Publish",
+      "Resource": "arn:aws:sns:REGION:ACCOUNT_ID:REPOSITORY"
+    }
+  ]
+}
+{{< /highlight >}}
+
+The **Resource** field should match the ARN of the topic.
+
+{{</markdown>}}
+{{< /tab>}}
+
+{{< tab header="Webhooks with Azure DevOps" >}}
```

Comment:
do we support Azure webhooks out-of-the-box?
not sure we should include information about Bitbucket and other cloud providers as I expect all source code hosted on Azure. Is it correct?

Reply:
Why do we want to remove them ? The idea is to show users that they can configure Github, Gitlab, Bitbucket, Azure webhooks to enable automatic build triggering.

Reply:
the guide is not for GitHub or Gitlab users. They already host code sources on Azure

Reply:
Sorry, I am trying to understand. Are you saying that App Center users host their sources on Azure only? If so, it is possible to use App Center Github app for ex. 
Or I am missing your point here ?

Reply:
exactly, my point that majority (99%) readers use Azure for git repos

Reply:
got it, thanks. I'll update it.

Reply:
Done